www/search

```diff
@@ -1508,13 +1512,15 @@ else if ($term || $browse)
                         $ratingExtra = "<span class=details> ($ratingExtra)</span>";
                 }
 
+                $eagerness = ($i < 5) ? "eager" : "";
+
                 // show the item
                 echo "<p>";
                 if ($art) {
                     // display with art
                     echo "<table class=grid border=0 cellspacing=0 cellpadding=0>"
                         . "<tr><td>"
-                        . "<a href=\"viewgame?id=$id\">"
+                        . "<a class='$eagerness' href=\"viewgame?id=$id\">"
```

Comment:
mix of single and double-quotes in HTML (harmless, similar issue in other lines)